docs/enhancements/api-for-ignoring-validations.md

```diff
@@ -0,0 +1,132 @@
+---
+title: api-for-skipping-validations
+authors:
+  - "@pmaidmen"
+creation-date: 2023-01-08
+last-updated: 2023-01-08
+---
+
+# Allow validations to be ignored
+
+# Summary
+
+We need a new api that will allow us to ignore cluster/host validations, this will be for unsupported use.
+This API should work on a per cluster basis and will have it's own organisation level permissions to allow us to enable it for specific organisations.
+An event should be emitted whenever an installation is started and this feature has been used. Additionally a feature flag should be raised to indicate this usage also.
+Clusters containing ignored validations should not contribute to statistics as they are unsupported.
+Clusters containing ignored validations should not be added to triage as they are unsupported.
+# Motivation
+
+Some users have very specific HW that doesn't pass our validations and we want to allow them to proceed with installation.
+Sometimes we have bugs in our validations that block people from installing and we don't want our partners left in a position where they can't install.
+
+### Goals
+
+- Allow validations to be bypassed so that installation may proceed.
+
+### Non-Goals
+
+- This is not intended for broad customer use and is not a "go to" solution for every validation problem
+- This should not add anything to the cluster API, this is unsupported and should not show up when a cluster is queried
+  - Instead there will be a specific API
+
+### The API
+This will consist of two endpoints.
+
+GET /v2/clusters/{cluster_id}/ignored-validations
+
+Will return the ignored host and cluster validations as a pair of JSON string arrays
+
+```
+{
+  "IgnoredHostValidations": "[\"has-memory-for-role\", \"has-cpu-cores-for-role\"]"
```

Comment:
```suggestion
  "ignored_host_validations": "[\"has-memory-for-role\", \"has-cpu-cores-for-role\"]"
```
or at least that's what I seeing in the swagger as the convention for field names in the response